Node Alpine Support

**Issue**
I've been trying to setup Ponder inside of docker using node alpine and it's been crashing on startup with this error `Cannot find module '@ast-grep/napi-linux-x64-musl'` it's an issue with `ast-grep`. Workaround for the time being is to just use an OS that uses glibc over msul libc.

**Fix**
This was fixed in this PR https://github.com/ast-grep/ast-grep/pull/944 and should be available for use in `0.19.2` and then I think alpine would work no problem if Ponder can update to it 🙏

--- a/.changeset/long-cups-grow.md
+++ b/.changeset/long-cups-grow.md
@@ -1,0 +1,5 @@
+---
+"@ponder/core": patch
+---
+
+Fixed Docker compatibility issues by upgrading ast-grep.

--- a/.changeset/long-cups-grow.md
+++ b/.changeset/long-cups-grow.md
@@ -2,4 +2,4 @@
 "@ponder/core": patch
 ---
 
-Fixed Docker compatibility issues by upgrading ast-grep.
+Updated ast-grep to 0.19.3 which fixed support for running Ponder on Alpine Linux-based Docker images.

--- a/packages/core/package.json
+++ b/packages/core/package.json
@@ -42,7 +42,7 @@
     }
   },
   "dependencies": {
-    "@ast-grep/napi": "^0.17.1",
+    "@ast-grep/napi": "^0.19.3",
     "@babel/code-frame": "^7.23.4",
     "abitype": "^0.10.2",
     "async-mutex": "^0.4.0",

